src/zkey/pkey.h

```diff
@@ -43,6 +43,31 @@ struct tokenheader {
 #define TOKEN_VERSION_CLEAR_KEY		0x02
 #define TOKEN_VERSION_EP11_AES		0x03
 
+/* Some ECC related constants */
+#define EC_PUBLEN_P256                   64
+#define EC_PUBLEN_P384                   96
+#define EC_PUBLEN_P521                  132
+#define EC_PUBLEN_ED25519                32
+#define EC_PUBLEN_ED448                  57
+
+#define EC_PRIVLEN_P256                  32
+#define EC_PRIVLEN_P384                  48
+#define EC_PRIVLEN_P521                  66
+#define EC_PRIVLEN_ED25519               32
+#define EC_PRIVLEN_ED448                 57
+
+#define EC_BITLEN_P256                  256
+#define EC_BITLEN_P384                  384
+#define EC_BITLEN_P521                  521
+#define EC_BITLEN_ED25519               255
```

Comment:
is this really 255 ? We use 256 at other places, but I am not sure what is the correct one.....

Reply:
Yes indeed. I have it from the CCA 7.3 book, chapter ECC key token, Table 345. Supported Edwards elliptic curves: Size of prime p in bits (key length): 

> 255    1.3.101.112     id-Ed25519    Curve25519